### ISSUE TITLE
Unknown arguments are swallowed

### ISSUE THREAD

**Issue by tingerrr** [andreasKroepelin/lovelace#13]
Any unknown arguments are swallowed, presumably because an argument sink is used.
```typst
#import "@preview/lovelace:0.3.0": pseudocode-list
#pseudocode-list(title: [My Algorithm], dosnt-exist: 9)[]
```

This means a that a type while refactoring can go unnoticed, resulting in incorrect output.

**Comment by andreasKroepelin** [comment]
Good point. For future reference, a check that `children.named()` is empty has to be added here:
https://github.com/andreasKroepelin/lovelace/blob/main/lovelace.typ#L121-L122
